### DISCUSSION TITLE
Dropdown menu position

### DISCUSSION THREAD

**Discussion by makayasa** [danvick/flutter_form_builder#1335] [General]
Can you made an option to customize dropdown menu position 

from this:
![image](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/52215902/bb94a0ea-1896-4ab2-ada3-0e02c7377935)

to something like this:
![image](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/52215902/83ada055-0d76-417e-8d04-61d51737a1fc)

**Comment by deandreamatias** [comment]
I think that no. This package use the default component of Material design, created by Flutter

**Reply by deandreamatias** [reply]
@makayasa  Hi. This still happens?
